terraform/alerting/alerts.tf

```diff
@@ -298,3 +298,37 @@ resource "google_monitoring_alert_policy" "HumanDecryptedValue" {
 
   notification_channels = [for x in values(google_monitoring_notification_channel.paging) : x.id]
 }
+
+resource "google_logging_metric" "export_file_downloaded" {
+  count = var.capture_export_file_downloads ? 1 : 0
+
+  name    = "export_file_downloaded"
+  project = var.project
+  filter  = <<EOT
+resource.type="http_load_balancer"
+httpRequest.requestUrl=~"/index.txt$"
```

Comment:
This needs to be parameterized by the folder (for per-country active counts)

Reply:
It is. It's named `path`, but I can change the name.